EIPS/eip-1845.md

```diff
@@ -0,0 +1,67 @@
+---
+eip: <to be assigned>
+title: Fork Name Standarts
+author: DimitryKh (dimitry@ethereum.org)>
+discussions-to: <URL>
```

Comment:
Can't merge this without a GitHub-issue or forum-thread to discuss any changes after the Draft has been merged.

Reply:
https://github.com/ethereum/retesteth/issues/23